modules/phase_field/test/tests/GrandPotentialPFM/GrandPotentialAnisotropy.i

```diff
@@ -0,0 +1,282 @@
+    D_name = Dchi
+    args = ''
+  [../]
+  [./coupled_etaa0dot]
+    type = CoupledSwitchingTimeDerivative
+    variable = w
+    v = etaa0
+    Fj_names = 'rhoa rhob'
+    hj_names = 'ha   hb'
+    args = 'etaa0 etab0'
+  [../]
+  [./coupled_etab0dot]
+    type = CoupledSwitchingTimeDerivative
+    variable = w
+    v = etab0
+    Fj_names = 'rhoa rhob'
+    hj_names = 'ha   hb'
+    args = 'etaa0 etab0'
+  [../]
+[]
+
+
+# enable_jit set to false in many materials to make this test start up faster.
+# It is recommended to set enable_jit = true or just remove these lines for
+# production runs with this model
+[Materials]
+  [./ha]
+    type = SwitchingFunctionMultiPhaseMaterial
+    h_name = ha
+    all_etas = 'etaa0 etab0'
+    phase_etas = 'etaa0'
+  [../]
+  [./hb]
+    type = SwitchingFunctionMultiPhaseMaterial
+    h_name = hb
+    all_etas = 'etaa0 etab0'
+    phase_etas = 'etab0'
+  [../]
+  [./omegaa]
+    type = DerivativeParsedMaterial
+    args = 'w'
+    f_name = omegaa
+    material_property_names = 'Vm ka caeq'
+    function = '-0.5*w^2/Vm^2/ka-w/Vm*caeq'
+    derivative_order = 2
+    enable_jit = false
+  [../]
+  [./omegab]
+    type = DerivativeParsedMaterial
+    args = 'w'
+    f_name = omegab
+    material_property_names = 'Vm kb cbeq'
+    function = '-0.5*w^2/Vm^2/kb-w/Vm*cbeq'
+    derivative_order = 2
+    enable_jit = false
+  [../]
+  [./rhoa]
+    type = DerivativeParsedMaterial
+    args = 'w'
+    f_name = rhoa
+    material_property_names = 'Vm ka caeq'
+    function = 'w/Vm^2/ka + caeq/Vm'
+    derivative_order = 2
+    enable_jit = false
+  [../]
+  [./rhob]
+    type = DerivativeParsedMaterial
+    args = 'w'
+    f_name = rhob
+    material_property_names = 'Vm kb cbeq'
+    function = 'w/Vm^2/kb + cbeq/Vm'
+    derivative_order = 2
+    enable_jit = false
+  [../]
+  [./kappaa]
+    type = InterfaceOrientationMultiphaseMaterial
+    kappa_name = kappaa
+    dkappadgrad_etaa_name = dkappadgrad_etaa
+    d2kappadgrad_etaa_name = d2kappadgrad_etaa
+    etaa = etaa0
+    etab = etab0
+  [../]
+  [./kappab]
+    type = InterfaceOrientationMultiphaseMaterial
+    kappa_name = kappab
+    dkappadgrad_etaa_name = dkappadgrad_etab
+    d2kappadgrad_etaa_name = d2kappadgrad_etab
+    etaa = etab0
+    etab = etaa0
+  [../]
+  [./const]
+    type = GenericConstantMaterial
+    prop_names =  'kappa_c   L   D    chi  Vm   ka    caeq kb    cbeq  gab mu'
+    prop_values = '0         1.0 1.0  0.1  1.0  10.0  0.1  10.0  0.9   4.5 10.0'
+  [../]
+  [./Mobility]
+    type = DerivativeParsedMaterial
+    f_name = Dchi
+    material_property_names = 'D chi'
+    function = 'D*chi'
+    derivative_order = 2
+    enable_jit = false
+  [../]
+[]
+
+[Preconditioning]
+  [./SMP]
+    type = SMP
+    full = true
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  scheme = bdf2
+  petsc_options_iname = '-pc_type -ksp_gmres_restart -sub_pc_type -pc_asm_overlap'
+  petsc_options_value = 'asm      31                  lu           1'
```

Comment:
asm/lu is not a very scalable preconditioner for production runs. I suggest using asm/ilu and adjustement of pc_factor_levels until it converges well.

Reply:
Oh, right, this is not an example but a test.

Reply:
@dschwen Hi Daniel. The execution time of this test is below 2s right now. Thank you.

Reply:
Hi Daniel, I run the test on my desktop computer with single core. I will reduce the test time further.

Reply:
@dschwen Hi Daniel. The test input file was modified again so that the test time is down to 1s on my desktop computer. Thank you.

Reply:
@dschwen Hello Daniel. Do you have any further comments? If everything looks good to you, could you activate all the tests? Thank you very much. Dehao